ci-operator/step-registry/cucushift/upgrade/hypershift/control-plane/toimage/cucushift-upgrade-hypershift-control-plane-toimage-commands.sh

```diff
@@ -100,8 +100,8 @@ echo "Compare MAIN version"
 set -x
 TARGET_VERSION="$(oc adm release info "${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}" -ojsonpath='{.metadata.version}')"
 TARGET_MAIN_VERSION="$(echo "$TARGET_VERSION" | cut -d '.' -f 1-2)"
-SOURCE_MAIN_VERSION="$(oc get clusterversion --no-headers | awk '{print $2}' | cut -d '.' -f 1-2)"
-echo "TARGET_MAIN_VERSION: $TARGET_MAIN_VERSION , SOURCE_MAIN_VERSION:$SOURCE_MAIN_VERSION"
+SOURCE_MAIN_VERSION="$(KUBECONFIG="${SHARED_DIR}/kubeconfig" oc get clusterversion --no-headers | awk '{print $2}' | cut -d '.' -f 1-2)"
```

Comment:
is there any problem using the default kubeconfig? I'm unsure the impact on the existing jobs

Reply:
The MC may not be an OCP cluster. Also we should be checking the guest cluster for the SOURCE_MAIN_VERSION. 

There shouldn't be any impact on existing jobs as the health checks in the same file below are currently performed against the guest: 
https://github.com/openshift/release/blob/0355ef9cba8eeaace8c4ab6337c61f61778a477e/ci-operator/step-registry/cucushift/upgrade/hypershift/control-plane/toimage/cucushift-upgrade-hypershift-control-plane-toimage-commands.sh#L137